hack/tools/release/notes.go

```diff
@@ -60,6 +60,9 @@ var (
 
 	fromTag = flag.String("from", "", "The tag or commit to start from.")
 
+	since = flag.String("since", "", "Include commits starting from this date. Format: 2023-02-06")
+	until = flag.String("until", "", "Include commits up until this date. Does NOT include commits during this date. Format: 2023-02-06")
```

Comment:
```suggestion
	until = flag.String("until", "", "Include commits up until this date. Does NOT include commits at this date. Format: 2023-02-06")
```
nit: I assume at this day?

Does this only support Year-Month-Day?

Can we add an example that makes it clear if this is Year-Month-Day or Year-Day-Month?

(not sure if Year-Day-Month is a thing, but Month-Day-Year exists, so I guess everything is possible)

Reply:
Git seems to accept a lot of interesting formats so maybe just reference git?

Reply:
Sounds good!